tests/unit/test_workflow_files.py

```diff
@@ -124,11 +124,11 @@ def test_check_nested_dirs(tmp_run_dir: Callable):
 
 @pytest.mark.parametrize(
     'test_install_path, existing_install_path',
-[
-    (f'{"child/"*i}', '') for i in range(2, MAX_SCAN_DEPTH)
-] + [
-    ('', f'{"child/"*i}') for i in range(1, MAX_SCAN_DEPTH)
-]
+    [
+        (f'{"child/"*i}', '') for i in range(2, MAX_SCAN_DEPTH + 3)
+    ] + [
+        ('', f'{"child/"*1}')
+    ]
```

Comment:
I don't think there is a need to have more than 1 test of each. But do need to check the scan dept for the second
```suggestion
    [
        pytest.param(
            f'{"child/" * (MAX_SCAN_DEPTH + 3)}',
            '',
            id="Check parents (beyond max scan depth)"
        ),    
        pytest.param(
            '',
            f'{"child/" * MAX_SCAN_DEPTH}',
            id="Check children up to max scan depth"
        )
    ]
```